src/cryptsetup/cryptsetup.c

```diff
@@ -353,7 +353,8 @@ static char *disk_mount_point(const char *label) {
 
         /* Yeah, we don't support native systemd unit files here for now */
 
-        if (asprintf(&device, "/dev/mapper/%s", label) < 0)
+        device = strjoin("/dev/mapper/", label);
```

Comment:
strjoina() ?

Reply:
Or, path_join()?

Reply:
`path_join` isn't really necessary, we know that the first item ends with a slash. `strjoina` would be marginally faster, but since this function is slow anyway, I don't think it matter much.